createdisk.sh

```diff
@@ -163,7 +163,17 @@ EOF
    PRE_DOWNLOADED_ADDITIONAL_PACKAGES+=" qemu-user-static-x86"
 fi
 
-# Beyond this point, packages added to the ADDITIONAL_PACKAGES variable won’t be installed in the guest
+# install 9pfs binary from COPR repo so that it can be used to
+# set up 9p file sharing on Windows
+if [ "${SNC_GENERATE_WINDOWS_BUNDLE}" != "0" ]; then
+    ${SSH} core@${VM_IP} -- "sudo dnf -y copr enable mskvarla/9pfs"
+    ${SSH} core@${VM_IP} -- "mkdir -p ~/packages && dnf download --downloadonly --downloaddir ~/packages 9pfs --resolve"
+    ${SSH} core@${VM_IP} -- "sudo dnf -y copr disable mskvarla/9pfs"
+    PRE_DOWNLOADED_ADDITIONAL_PACKAGES+=" 9pfs"
+fi
+
+# Beyond this point, packages added to the ADDITIONAL_PACKAGES and PRE_DOWNLOADED_ADDITIONAL_PACKAGES
+# variables won’t be installed in the guest
```

Comment:
What is our plan around it, always going to use the copr or it is just for initial POC side? As per https://download.copr.fedorainfracloud.org/results/mskvarla/9pfs/srpm-builds/09226159/9pfs.spec file looks like we are just building the binary, can we migrate to container image for this which can be built on brew internally?